README.md

```diff
@@ -1,66 +1,88 @@
+[![codecov](https://codecov.io/gh/bringyourownideas/silverstripe-maintenance/branch/master/graph/badge.svg)](https://codecov.io/gh/bringyourownideas/silverstripe-maintenance)
 
-* Provides information about
- * available updates for composer packages,
- * known security issues of all installed packages, even dependencies of dependencies and
- * complete list of installed composer packages, including global packages and dependencies.
-* All information will be saved to the database as well as displayed in a model admin.
-* Scheduling of updates of the information
-
-
-## Source of the information
+## Overview
 
-The information is based on your composer files. So you need to have them available in the environment you plan to use this module. The modules below process the content of the composer files and check in suitable sources for information regarding your set up.
-
-The main functionality comes from these modules:
-
-* [SilverStripe Composer Security Checker](https://github.com/bringyourownideas/silverstripe-composer-security-checker "Check your SilverStripe application for security issues")
-* [SilverStripe Composer Update Checker](https://github.com/bringyourownideas/silverstripe-composer-update-checker "Check your SilverStripe application for available updates of dependencies.")
-* [SilverStripe Composer Versions](https://github.com/bringyourownideas/silverstripe-composer-versions "Provides your installed composer versions within your SilverStripe app, for review or other use cases.")
+The [SilverStripe Maintenance module](https://github.com/bringyourownideas/silverstripe-maintenance "Assists with the 
+maintenance of your SilverStripe application") reduces your maintenance related work.
 
+The module provides information about:
+* available updates for composer packages,
+* known security issues of all installed packages, even dependencies of dependencies and
+* complete list of installed composer packages, including global packages and dependencies.
+* All information will be saved to the database as well as displayed in a model admin.
+* Scheduling of updates of the information
 
-## Requirements and installation
+## Requirements and Installation
 
 ### Requirements
 
 * You require the composer.json and composer.lock files to be available and readible in the environment you plan to use this module. All information is based on these files.
-* Install at least one of the modules mentioned under "Source of the information". As a development dependency should be fine in most cases.
+* Install at least one of the modules mentioned under "Source of the information". A development dependency should be fine in most cases.
 * The queuedjob module is a dependency as the checks are scheduled using queuedjobs. This saves you time and work at the end.
 
+### Installation 
+ 
+Install the maintenance package:
+```
+composer require bringyourownideas/silverstripe-maintenance
+```
```

Comment:
Can we mention in here somewhere that `^1` should be used for SilverStripe 3 and `^2` for SilverStripe 4?